### ISSUE TITLE
Footer missing an h2 for meta links

### ISSUE THREAD

**Issue by roblav** [hmrc/play-ui#123]
An internal accessibility review found that our footer was missing an h2.
"Check that you are using the most recent code to generate the footer.  The H2 text ‘Support links’ looks to be missing - h2 class="govuk-visually-hidden"/>Support links h2/>
See GOV.UK design system https://design-system.service.gov.uk/components/footer/ the section ‘Footer with links to meta information’ seems to match the layout you have."
